registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/validator/PacketValidator.java

```diff
@@ -0,0 +1,17 @@
+package io.mosip.registration.processor.stages.validator;
+
+import io.mosip.registration.processor.core.abstractverticle.MessageDTO;
+import io.mosip.registration.processor.core.logger.LogDescription;
+import io.mosip.registration.processor.core.packet.dto.PacketMetaInfo;
+import io.mosip.registration.processor.core.status.util.TrimExceptionMessage;
+import io.mosip.registration.processor.core.util.IdentityIteratorUtil;
+import io.mosip.registration.processor.stages.dto.PacketValidationDto;
+import io.mosip.registration.processor.stages.exception.PacketValidatorException;
+import io.mosip.registration.processor.status.dto.InternalRegistrationStatusDto;
+
+public interface PacketValidator {
+
+	boolean validate(InternalRegistrationStatusDto registrationStatusDto, PacketMetaInfo packetMetaInfo,

```

Comment:
can u analyze little bit if we can restructure this little bit. if we need all the inputs, like if we can remove few trimMessage,MessageDTO,  identityIteratorUtil, packetValidationDto etc.
It will be better if we can restructure it since its not possible for external SI to understand all these inputs and it will lead to more confusion,